src/Symfony/Component/RateLimiter/FixedWindowLimit.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+namespace Symfony\Component\RateLimiter;
+
+
+final class FixedWindowLimit implements LimitInterface
```

Comment:
What is the reason of creating 2 limit classes? If you move `$this->windowOpeningDateTime->add(new \DateInterval(sprintf('P%sS', $this->interval)))` to the limiter, you can use one class for both right?

Reply:
At beginning, I was doing operation in Limit because I hadn't understood the goal to return Limit on consume (and not to inject it in Limiter)